frontend/src/styles/modalProfile.css

```diff
@@ -1,16 +1,128 @@
-.modal-bio{
-    font-family:'Roboto', sans-serif;
+.modal-bio {
     margin: 0 15px;
     opacity: 0;
     transition: opacity 1.1s ease-in;
     font-size: 1.3rem;
+    overflow-y: auto;
+}
+
+h2 {
+    color: #fff;
+    font-family:'Playfair display', serif;
+    font-weight: 900;
+    letter-spacing: .05em;
 }
 
 .modal-bio.fade {
     opacity: 1;
 }
+.modal-general-div {
+    margin: 0;
+}
 
-p {
-    font-family:'Roboto', sans-serif;
-    font-size: 1.7rem;
+.modal-flex-div {
+    display: flex;
+    flex-wrap: wrap;
 }
+
+.modal-flex-section {
+    max-width: 48%;
+    padding: 0 10px;
```

Comment:
This 10px padding is putting "Experience" and "Skills" slightly nested under the "About" header. I vote we either knock it down to 0px to be inline with the other sections or increase it to something like 25px to make it clear we intentionally want it as sub-sections under "About"

Reply:
Great catch. 
I fixed this so that these sections have the same spacing as the other sections.